### ISSUE TITLE
Loooong compile times with 0.7.1 compared to 0.6.12

### ISSUE THREAD

**Issue by MrPowerGamerBR** [Kotlin/kotlinx.html#147]
This is more of an PSA than anything else, this can be closed after the compiler bug is fixed, I decided to post this issue to avoid other developers scratching their heads thinking "wow kotlin surely is slow, it is taking waaaay too long to compile it"

Recently I've updated Kotlinx.html on one of my projects and... oh man, the compile times are absurd compared to 0.6.12.

I was using kotlinx.html with Kotlin scripting engine, and I've noticed that the compile times went from 30s at worst to *30 minutes*.

This is due to https://github.com/Kotlin/kotlinx.html/commit/0f24cfeed22375fd01c3118456e9620e61dea04b

This also seems to be noticed by other developers at Kotlin's slack: https://kotlinlang.slack.com/archives/C7L3JB43G/p1587514495067900

YouTrack compiler issue: https://youtrack.jetbrains.com/issue/KT-38489

tl;dr: Use 0.6.12 while the bug isn't fixed!

**Comment by hhariri** [comment]
This should be resolved as a consequence of the compiler issue resolved. https://youtrack.jetbrains.com/issue/KT-38489
